Documentation/source/Changelog.md

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+3.3.5 (2018-07-06)
+------------------
+
+-   Added flag for Drag&Drop step in "Open with" tutorial (via `GiniConfiguration.shouldShowDragAndDropTutorial` property).
```

Comment:
Articles man!

> Added a flag for Drag&Drop in the "Open with" tutorial (via the `GiniConfiguration.shouldShowDragAndDropTutorial` property).

Reply:
Added **a** flag for **the** Drag&Drop step
or
Added **a** flag for Drag&Drop <strike>step</strike>